meteor/client/lib/reactiveData/reactiveDataHelper.ts

```diff
@@ -161,8 +161,10 @@ export abstract class WithManagedTracker {
 		}, 2000) // wait for a couple of seconds, before unsubscribing
 	}
 
-	protected subscribe(sub: PubSub, ...args: any[]) {
-		this._subs.push(Meteor.subscribe(sub, ...args))
+	protected subscribe(sub: PubSub, ...args: any[]): Meteor.SubscriptionHandle {
```

Comment:
How about making the API of `WithManagedTracker` similar to `MeteorReactComponent` and having `subscriptionsReady` method?

Reply:
Sounds good, and works just as well https://github.com/tv2/tv-automation-server-core/commit/a3859077bacee156470abf801efd1f58b9181be6